### PR TITLE
dawn: fix network packet format to be host-agnostic

### PR DESCRIPTION
**Pull request by dwmw2**
I tried Dawn and it whined at me because it didn't understand its
own packets from a different-endian host.

```
Mon Jun  8 10:49:12 2020 daemon.err dawn[19742]: not complete msg, len:308, expected len:872480768
Mon Jun  8 10:49:12 2020 daemon.err dawn[19742]: not complete msg, len:708, expected len:3288465408
```
https://github.com/berlin-open-wireless-lab/DAWN/issues/92

Signed-off-by: David Woodhouse <dwmw2@infradead.org>

Maintainer: @PolynomialDivision 
Compile tested: mipsel Asus RT-AC85P, OpenWRT master.
                          mipseb WNDR3800, OpenWRT 19.07
                          mipseb BT Home Hub 5a (Lantiq XRX2xx), OpenWRT 19.07

Run tested: Observe the warning go away and the mipsel machine actually being able to see data from the other two.
Description:

### CODE REVIEW

File: net/dawn/Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dawn
 PKG_SOURCE_DATE:=2020-06-06
```
Comment (PolynomialDivision) [inline net/dawn/Makefile:8]: please update the date

File: net/dawn/Makefile [hunk 2]
```diff
@@ -6,7 +6,7 @@ include $(TOPDIR)/rules.mk
...
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```
Comment (PolynomialDivision) [inline net/dawn/Makefile:9]: this has to stay at 1

**Comment by PolynomialDivision** [comment]
I think adding a patch file is unnecessary, if u update your commit msg accordingly I will merge it and then u can directly update the package version. :) I already tested it.

**Comment by dwmw2** [comment]
OK, as soon as it's merged into the upstream DAWN repo I can just update to use a new snapshot.

**Comment by PolynomialDivision** [comment]
Merged in https://github.com/berlin-open-wireless-lab/DAWN/commit/050c1d7fed624161d29b09f1b46708d342496012

**Comment by dwmw2** [comment]
Package updated to new snapshot instead of patching. Thanks.

**Comment by PolynomialDivision** [comment]
lgtm

**Comment by dwmw2** [comment]
Want me to push it for you?

**Comment by PolynomialDivision** [comment]
> Want me to push it for you?

You have commit access? Then do it. :) I don't have commit access.

**Comment by dwmw2** [comment]
Done; thanks.,

**Pull request closed without merging**